src/main/java/Deadline.java

```diff
@@ -0,0 +1,41 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+// partial solution copied from iP
```

Comment:
A Javadoc comment would be more appropriate here since the reader might not know what you are referring to